src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonPropertyInfo.cs

```diff
@@ -732,25 +732,6 @@ private bool NumberHandingIsApplicable()
 
         internal abstract object? GetValueAsObject(object obj);
 
-#if DEBUG
```

Comment:
Removes a number of debug traces that were added a few years ago while a bug was being investigated.